frontend/packages/dev-console/src/components/topology2/components/nodes/__tests__/ApplicationGroup.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { findLowestPoint } from '../ApplicationGroup';
+
+describe('ApplicationGroup', () => {
+  it('should return the lowest point', () => {
+    expect(findLowestPoint([[0, 0, 0]])).toEqual([0, 0, 0]);
+    expect(findLowestPoint([[100, 10, 1000], [200, 30, 2000], [300, 20, 3000]])).toEqual([
```

Comment:
Not sure I understand - `findLowestPoint` gets you the mid-point?

Reply:
`findLowestCenterPoint` maybe?

Reply:
It's supposed to help find the lowest point where to place the node label. Note that this is _internal_ to the component itself and only exposed for testing.

Reply:
went with `computeLabelLocation` :)